Makefile

```
@@ -145,3 +145,11 @@ pinpoint-gateway:
 help: Makefile
 	@echo " Choose a command run in pinpoint:"
 	@sed -n 's/^##//p' $< | column -t -s ':' |  sed -e 's/^/ /'
+
+.PHONY: gen
+## gen: Generates API code from swagger tool 
+gen: ( cd api ; swagger generate server -t gen -f ./swagger/swagger.yml -A spotter )
```

also this isn't how Makefiles work - the first line declares other `make` targets to run. So this command errors out like so:

```
❯ make gen
make: *** No rule to make target `(', needed by `gen'.  Stop.
```

because it's trying to run `make (` and then `make cd` and then `make api` etc.

```suggestion
gen:
    ( cd api ; swagger generate server -t gen -f ./swagger/swagger.yml -A spotter )
```

---

Also, this target is already in use, so it breaks CI: https://travis-ci.com/ubclaunchpad/pinpoint/jobs/178453301#L633

https://github.com/ubclaunchpad/pinpoint/blob/master/Makefile#L72

^^ that's also an example of how the chained Make targets work

---

try to follow the convention there 😄

---

hehehehe good to know